suricata.yaml.in

```diff
@@ -113,7 +113,7 @@ outputs:
 
       # Community Flow ID
       # Adds a 'community_id' field to EVE records. These are meant to give
-      # a records a predictable flow id that can be used to match records to
+      # records a predictable flow ID that can be used to match records to
       # output of other tools such as Bro.
```

Comment:
Should change this to Zeek while in here.

Reply:
Zeek (Bro) would be my suggestion.